### PR TITLE
feat: add codemods for `number.*` polyfills

### PR DESCRIPTION
**Pull request by xeho91**
Nothing much to do with those polyfills packages. Except removing the possible imports.

Luckily they're not popular _(based on downloads)_.

### CODE REVIEW

**Comment by xeho91** [comment]
Related: https://github.com/es-tooling/module-replacements/pull/110

File: test/fixtures/number.isfinite/case-1/before.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const assert = require("assert");
```
Comment (thepassle) [inline test/fixtures/number.isfinite/case-1/before.js:1]: is this `before` file correct? I dont see an import for `number.isfinite`

File: test/fixtures/number.isinteger/case-1/before.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const assert = require("assert");
```
Comment (thepassle) [inline test/fixtures/number.isinteger/case-1/before.js:1]: im also missing the import for `number.isinteger` here

**Comment by thepassle** [comment]
Thanks for the PRs! Can you double check the `before` fixtures? I'm missing imports for the packages that should be removed. Thanks!

**Comment by xeho91** [comment]
> Thanks for the PRs! Can you double check the `before` fixtures? I'm missing imports for the packages that should be removed. Thanks!

You're right, I fixed em.

G**ood to note:** I am not sure if I got the imports right. Because to be honest, I struggle to understand the complexity of the polyfill system 🤷

**Comment by thepassle** [comment]
Yeah I think this is fine for now, the es-shims polyfill api or whatever is pretty weird, we can improve on it in the future if it turns out to be a problem. The largest uses of these packages are fairly straightforward; import function, call function.

Thanks for the contribution!

**Pull request merged**